### DISCUSSION TITLE
rime插件下，同步报错，是rime/.temp.userdb下文件权限导致的

### DISCUSSION THREAD

**Discussion by zhichengroup** [fcitx5-android/fcitx5-android#490] [General | 讨论]
rime插件下，同步时报错，是rime/.temp.userdb下文件权限导致的。
这个issues下也有人有相同情况[issues/452/2016578079](https://github.com/fcitx5-android/fcitx5-android/issues/452#issuecomment-2016578079)，不确定是软件bug还是系统bug，android自带的文件管理器也删不掉 ，但在fcitx-android中备份-卸载-重新导入就无了。
本来怀疑是和linux之间同步时造成的文件权限问题，目前看好像不是。

**Comment by GD-Slime** [comment]
我也遇到了这个情况。备份卸载也处理不掉这个临时文件夹。导致同步必失败。

**Reply by zhichengroup** [reply]
在文件管理器中把`.temp.userdb` 重命名城`.temp.userdb1`，再同步试试

**Comment by closure11** [comment]
我也有这个问题，但是我在rime文件夹里找不到.temp.userdb(已经打开隐藏文件显示)，那要怎么删呢

看日志确实和大家一样都是读这个数据库报的错
```
E/rime.fcitx-rime(11862): Error opening db '.temp': IO error: lock /storage/emulated/0/Android/data/org.fcitx.fcitx5.android/files/data/rime/.temp.userdb/LOCK: already held by process
```

**Reply by zhichengroup** [reply]
是原生Android文件管理器吗？进入rime文件夹之后，点击右上角三个点，有个显示隐藏的文件，就有了。
建议用小企鹅更新器把主程序和rime插件更新到最新版本，不清楚是不是上游修复了，我已经好久没遇到这个问题了。
